packages/amplify-provider-awscloudformation/src/configuration-manager.ts

```diff
@@ -454,12 +453,13 @@ async function validateConfig(context: $TSContext) {
         awsConfigInfo.configValidated = true;
       }
     } else {
-      awsConfigInfo.configValidated = awsConfigInfo.config.accessKeyId
-        && awsConfigInfo.config.accessKeyId !== constants.DefaultAWSAccessKeyId
-        && awsConfigInfo.config.secretAccessKey
-        && awsConfigInfo.config.secretAccessKey !== constants.DefaultAWSSecretAccessKey
-        && awsConfigInfo.config.region
-        && awsRegions.regions.includes(awsConfigInfo.config.region);
+      awsConfigInfo.configValidated =
```

Comment:
it also occurred to me that moving the && to a newline is also more consistent with our ternary formatting where the ? and : go on a newline

Reply:
see above